tests/trading/test_makeOrder.py

```diff
@@ -155,7 +155,7 @@ def test_makeOrder_failure(contractsFixture):
 
     # makeOrder exceptions (pre-placeBid/placeAsk)
     with raises(TransactionFailed):
-        makeOrder.publicMakeOrder(ASK, fix('1'), fix('0.6'), market.address - 1, YES, 0, 0, 42, sender=tester.k1)
+        makeOrder.publicMakeOrder(ASK, fix('1'), fix('0.6'), market.address, YES, 0, 0, 42, sender=tester.k1)
```

Comment:
This was failing for _multiple_ reasons where it was placed in the test. I will move it to a more appropriate location.